src/devices/inovelli.ts

```diff
@@ -2178,46 +1997,28 @@ const definitions: DefinitionWithExtend[] = [
             tzLocal.inovelli_parameters(VZM31_ATTRIBUTES),
             tzLocal.inovelli_parameters_readOnly(VZM31_ATTRIBUTES),
         ],
-        fromZigbee: [
-            fz.on_off,
-            fz.brightness,
-            fz.level_config,
-            fz.power_on_behavior,
-            fz.ignore_basic_report,
-            fz.electrical_measurement,
-            fz.metering,
-            fzLocal.inovelli(VZM31_ATTRIBUTES),
-        ],
+        fromZigbee: [fz.on_off, fz.brightness, fz.level_config, fz.power_on_behavior, fz.ignore_basic_report, fzLocal.inovelli(VZM31_ATTRIBUTES)],
         ota: ota.inovelli,
         configure: async (device, coordinatorEndpoint) => {
             const endpoint = device.getEndpoint(1);
-            await reporting.bind(endpoint, coordinatorEndpoint, ['seMetering', 'haElectricalMeasurement', 'genOnOff', 'genLevelCtrl']);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
             await reporting.onOff(endpoint);
 
             // Bind for Button Event Reporting
             const endpoint2 = device.getEndpoint(2);
             await reporting.bind(endpoint2, coordinatorEndpoint, ['manuSpecificInovelli']);
-            await endpoint.read('haElectricalMeasurement', ['acPowerMultiplier', 'acPowerDivisor']);
```

Comment:
Are these two lines needed? Or does this happen automatically now?

I'm concerned that the correct multipliers/divisors are not being used anymore since the power is being measured as 500-600w when it previously was showing up as 50-60w (before this change)

Reply:
To clarify @Koenkk, does this happen automatically when calling the `electricityMeter` modern extend?

```
await endpoint.read('haElectricalMeasurement', ['acPowerMultiplier', 'acPowerDivisor']);
await reporting.readMeteringMultiplierDivisor(endpoint);
```

I'm guessing it is not being applied since I'm seeing a 10x increase in power measurement after this code was removed.

Reply:
> To clarify @Koenkk, does this happen automatically when calling the electricityMeter modern extend?

Yes  it does: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/lib/modernExtend.ts#L1874

Reply:
Hmm that looks like it should be working... 

Any idea why the power is 10x higher than before these changes? I'm guessing I missed something.

Reply:
Could it be that device reports the wrong divisor? You should see what it responds in the configure in the debug log


See [this](https://www.zigbee2mqtt.io/guide/configuration/logging.html) on how to enable debug logging.

Reply:
Looks good, it reports the value here:

```
[2024-12-03 11:04:13] debug: 	z2m: Received Zigbee message from 'Kitchen Lights Load Switch', type 'readResponse', cluster 'haElectricalMeasurement', data '{"acPowerDivisor":10,"acPowerMultiplier":1}' from endpoint 1 with groupID 0
```

Reply:
I swear it wasn't working when I originally posted this comment. Just tested it again and now it's working fine...